_content/booklets/buddhist-perspective-on-time-and-space_hsingyun.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "The Buddhist Perspective on Time and Space"
+authors:
+  - hsingyun
+translator: "Amy Lam and Jeanne Tsai"
+editor: "Fo Guang Shan International Translation Center"
+external_url: "https://www.fgsitc.org/wp-content/uploads/2023/04/A17-The-Buddhist-Perspective-on-Time-and-Space-2019.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/1AvK3oc6NJbrcUKSqGWjl8YL4meq0RYCN/view?usp=drivesdk"
+course: cosmology
+tags:
+  - time
+  - space
+  - philosophy
+  - east-asia
+year: 1998
+publisher: "Fo Guang Shan International Translation Center"
+address: "Taiwan"
+pages: 32
```

Comment:
hmmm, I'd estimate 48 - 14 (front and back matter pages) = 34 (small pages) ÷ 2 (per A4) ≈ 17

BUT we know from [the previous PDF](https://drive.google.com/file/d/11XtAxxvLMK1bcweBUCwmLAjKLEyKTw5f/view?usp=drivesdk) that this is actually 18 letter-sized pages (with that wide left margin).  So my estimate wasn't too far off! 😊